### ISSUE TITLE
Plain text files are rendered a bit lower in the page

### ISSUE THREAD

**Issue by eksperimental** [elixir-lang/ex_doc#1484]
I expect the first line to be at the top of the page, but because it has not set Title,
the `<h1>` block is still there but empty.

![image](https://user-images.githubusercontent.com/9133420/149549391-7f4bac16-8342-4ef5-b25e-2309a775c379.png)

I think the `<h1>` block should be removed when no title is present.

**Comment by eksperimental** [comment]
This is a regular page.

![image](https://user-images.githubusercontent.com/9133420/149549673-591d1564-2086-43b6-9abf-6a24f17bc3e8.png)

**Comment by josevalim** [comment]
Yes, this is intentional. Otherwise, if your first line is a long line, it ends-up clobbering and conflicting with `</>` and the cog. You can explicitly set a title. I am also glad to receive a PR with different approaches, but for all purposes, it is meant to not conflict with the page layout. Thanks!

**Comment by eksperimental** [comment]
the title is set in the extra config.

```
extra: [
        "LICENSES/LICENSE.0BSD.txt": [filename: "license-0BSD", title: "BSD Zero Clause License"]
]
```

**Comment by josevalim** [comment]
Oh, so we are ignoring the title in those cases? Hrm, we should not. I will investigate.

**Comment by eksperimental** [comment]
Awesome, much better now.
Closing it since it is fixed in 2ce41968b676db04aee006abb1d8abc4ed72d14b
